package/yast2.spec

```diff
@@ -1,7 +1,7 @@
 #
 # spec file for package yast2
 #
-# Copyright (c) 2015 SUSE LINUX GmbH, Nuernberg, Germany.
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
```

Comment:
This is what spec-cleaner does automatically. Maybe @scarabeusiv can say something about this.